lib/api/v3/statuses/status_collection_representer.rb

```diff
@@ -12,8 +12,8 @@
 #
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; either status 2
-# of the License, or (at your option) any later status.
```

Comment:
Haha, someone did totally not c&p&replace code here xD

Chances are high that this someone, was actually me ^^